### PR TITLE
Update nav.adoc and Troubleshooting

### PR DESCRIPTION
**Pull request by voroyam**
add appliance trouble shooting section

This is kind of a backport of the changes I made in the appliance doc.

### CODE REVIEW

**Comment by mmattel** [comment]
@voroyam any reason for having items not in alphabetical order? Does not make sense to me...

**Comment by mmattel** [comment]
Q: with this PR, are we in sync with master or is a forward port necessary?

**Comment by voroyam** [comment]
> @voroyam any reason for having items not in alphabetical order? Does not make sense to me...

You mean in the appliance section? I think when I organized them I put them in order of importance or the order of frequency of the topic being needed for people from my point of view. Alphabetical order seems a good way to order it though. If this was really your suggestion then I think it's a good one.

**Comment by voroyam** [comment]
> Q: with this PR, are we in sync with master or is a forward port necessary?

Since I copy pasted the page from the master branch, this would make us in sync with the master branch :)

**Comment by voroyam** [comment]
This will fail until we backport the app settings in to this.

**Comment by settermjd** [comment]
> This will fail until we backport the app settings in to this.

How would a navigation change fail?

**Comment by voroyam** [comment]
well, I did not backport the app setting to the 10.3 branch but it's referenced in this document.

I think the fasted way to solve this is to remove the app settings reference.

**Comment by voroyam** [comment]
Now it should be good

**Comment by settermjd** [comment]
Thanks for the clarification, @voroyam.

File: modules/admin_manual/pages/appliance/troubleshooting.adoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+= Troubleshooting
+
+If you have encountered an issue, here is what support needs in order to get a quick resolution of your issue:
```
Comment (settermjd) [inline modules/admin_manual/pages/appliance/troubleshooting.adoc]: I suggest the sentence end with "_in order to provide a rapid resolution to your issue:_"

Comment (voroyam) [reply modules/admin_manual/pages/appliance/troubleshooting.adoc]: Sounds good

**Comment by mmattel** [comment]
occ command is not in sort order, same for hsm. This looks totally ugly and is hard to find.
A alphabetical sort order is a natural way of sorting and finding while putting things is personal sort order based on personal preferences is uncommon and may not fit for the next reader. If needed, make chapters/sections/submenues to group things.

Could we agree that the source is master and we backport / align to master.
Any changes needed in any release shoul de made in master and than backported and only in a rare cases there should be a forward port. This is the only ways of being future proof - just my 2c

Based on that:
create a PR which backports missing things from master to 10.3
create a PR`s that fixes/changes if necessary in master and backport these changes to 10.3

**Comment by settermjd** [comment]
> occ command is not in sort order, same for hsm. This looks totally ugly and is hard to find.
> A alphabetical sort order is a natural way of sorting and finding while putting things is personal sort order based on personal preferences is uncommon and may not fit for the next reader. If needed, make chapters/sections/submenues to group things.

@mmattel, I agree with you, but changing the sort order of the occ commands TOC is for a separate PR.

> 
> Could we agree that the source is master and we backport / align to master.
> Any changes needed in any release shoul de made in master and than backported and only in a rare cases there should be a forward port. This is the only ways of being future proof - just my 2c
> 

Agreed.

> Based on that:
> create a PR which backports missing things from master to 10.3
> create a PR`s that fixes/changes if necessary in master and backport these changes to 10.3

**Comment by mmattel** [comment]
> changing the sort order of the occ commands TOC is for a separate PR

I am not talking about changing the sort order for the TOC for occ, but the location (sort order) of the occ document which has bee moved in this PR.

**Comment by settermjd** [comment]
> > changing the sort order of the occ commands TOC is for a separate PR
> 
> I am not talking about changing the sort order of the TOC for occ, but the location (sort order) of the occ document which has bee moved in this PR which is now out of sort order, same for hsm

On having a second look, I see what you're saying. Well spotted.

**Comment by settermjd** [comment]
I've updated the location of the occ command's nav element.

**Pull request merged**